torch/csrc/jit/tensorexpr/codegen.h

```diff
@@ -182,6 +186,11 @@ TORCH_API std::unique_ptr<CodeGen> CreateCodeGen(
     const std::vector<CodeGen::BufferArg>& params,
     at::Device device = at::kCPU);
 
+class TORCH_API GenericIntrinsicsExpander : public IRMutator {
```

Comment:
Nit: we're creating an entire class (+ derived from it) just to redefine a single function. We could've get away with just a single free-standing function here.

Reply:
This class and function is invoked implicity through accept_mutator. Unless we come up with more magic wrappers here, there is no easy way to call into this in our current infrastructure with a free standing function.